extensions/amp-story/0.1/amp-story.js

```diff
@@ -79,6 +79,16 @@ const FULLSCREEN_THRESHOLD = 1024;
 /** @private @const {number} */
 const DESKTOP_THRESHOLD = 768;
 
+/**
+ * @private @const {string}
+ */
+const AUDIO_MUTED_ATTRIBUTE = 'muted';
+
+/**
+ * @private @const {string}
+ */
+const AUDIO_UNMUTED_ATTRIBUTE = 'unmuted';
```

Comment:
We should only need a single boolean attribute. Jon suggested `muted`.

Reply:
My bad, misread the description. 
Done now